src/Pages/ContactPage.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import {
+  Container,
+  Typography,
+  Box,
+  Paper,
+  makeStyles,
+} from "@material-ui/core";
+import Footer from "../Components/Footer";
+
+const useStyles = makeStyles({
+  bg: {
+    backgroundColor: "#2D3047",
+  },
+});
+
+const ContactPage = () => {
+  const classes = useStyles();
```

Comment:
Do we need this here? It's currently unused, but perhaps we could use it for any custom styling on the page?

Reply:
Yes I was beginning to move things up there last night.